src/main/java/org/agbrothers/englishbot/repository/UserRepository.java

```diff
@@ -1,9 +1,19 @@
 package org.agbrothers.englishbot.repository;
 
 import org.agbrothers.englishbot.entity.User;
-import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
-public interface UserRepository extends CrudRepository <User, Long> {
+public interface UserRepository extends JpaRepository<User, Long> {
+
+    @Query("SELECT chatId from User")
+    List<String> getChatIdsUser();
+
+    @Query("SELECT user FROM User user WHERE user.chatId = :chatId")
+    User getUserByChatId(@Param("chatId") String chatId);
```

Comment:
I believe you should not use `@Query` here, since `JpaRepository` API can perform the select by correctly named method